Sources/AppStoreConnectCLI/Model/BetaGroup.swift

```diff
@@ -6,18 +6,39 @@ import Foundation
 import SwiftyTextTable
 
 struct BetaGroup: TableInfoProvider, ResultRenderable, Equatable {
-    let appBundleId: String?
-    let appName: String?
-    let groupName: String?
-    let isInternal: Bool?
-    let publicLink: String?
-    let publicLinkEnabled: Bool?
-    let publicLinkLimit: Int?
-    let publicLinkLimitEnabled: Bool?
-    let creationDate: Date?
+    let appId: String
+    var appBundleId: String?
+    var appName: String?
+    var groupName: String?
+    var isInternal: Bool?
+    var publicLink: String?
+    var publicLinkEnabled: Bool?
+    var publicLinkLimit: Int?
+    var publicLinkLimitEnabled: Bool?
+    var creationDate: String?
+
+    init(appId: String) {
+        self.appId = appId
+    }
+
+    mutating func update(with attributes: AppStoreConnect_Swift_SDK.App.Attributes?) {
```

Comment:
Could these be extensions?  We've been putting anything that interacts with `AppStoreConnect_Swift_SDK` types in extensions.  My feeling has generally been that we should try and keep the `AppStoreConnect_Swift_SDK` type interactions seperate from the main model struct definitions.